src/server/chromium/crConnection.ts

```diff
@@ -100,7 +100,7 @@ export class CRConnection extends EventEmitter {
     for (const session of this._sessions.values())
       session._onClosed(browserDisconnectedLogs);
     this._sessions.clear();
-    Promise.resolve().then(() => this.emit(ConnectionEvents.Disconnected));
+    void Promise.resolve().then(() => this.emit(ConnectionEvents.Disconnected));
```

Comment:
I don't quite like these voids for no real reason 😄

Reply:
For these `void Promise.resolve().then(` calls we can either introduce a helper function that will do `.catch` internally or can add `.catch()` inline which is silly. Which way you like better?

Reply:
I like neither 😄